itou/templates/approvals/detail.html

```diff
@@ -7,7 +7,7 @@
     {% include "layout/previous_step.html" with back_url=back_url only %}
 {% endblock %}
 
-{% block title_content %}<h1>{{ approval.user.get_full_name }}</h1>{% endblock %}
+{% block title_content %}<h1>Salarié : {{ approval.user.get_full_name }}</h1>{% endblock %}
```

Comment:
Me semble que le cas peux pas se produire mais vu qu'on a la variable et qu'on l’utilise pour les _includes_
```suggestion
{% block title_content %}<h1>Salarié : {{ approval.user.get_full_name|mask_unless:can_view_personal_information }}</h1>{% endblock %}
```

Reply:
Je me le note pour une autre PR.